### PR TITLE
Use new oc_configmap to pay back tech debt

### PR DESCRIPTION
**Pull request by themurph**

### CODE REVIEW

File: ansible/roles/openshift_online_ha_proxy/tasks/main.yml [hunk 1]
```diff
@@ -14,21 +14,21 @@
...
-# when oc attempted to read the temporary file created by oc_obj.
-# We have documented the work necessary to address these problems long term. In the interim, the command
-# module will be used.
-
-- name: Delete the configmap if it exists
-  command: >
-    oc delete configmap customrouter -n default --ignore-not-found
+- name: Delete existing config map if necessary
+  oc_configmap:
+    state: absent
```
Comment (kwoodson) [inline ansible/roles/openshift_online_ha_proxy/tasks/main.yml]: Why the delete here?

File: ansible/roles/openshift_online_ha_proxy/tasks/main.yml [hunk 2]
```diff
@@ -14,21 +14,21 @@
...
-# of the complex content in the customrouter configmap.
-# An attempt was made to use oc_obj to create the configmap with the necessary content, but we ran into errors
-# when oc attempted to read the temporary file created by oc_obj.
-# We have documented the work necessary to address these problems long term. In the interim, the command
-# module will be used.
-
-- name: Delete the configmap if it exists
-  command: >
-    oc delete configmap customrouter -n default --ignore-not-found
+- name: Delete existing config map if necessary
```
Comment (jupierce) [inline ansible/roles/openshift_online_ha_proxy/tasks/main.yml]: @themurph I'm assuming @kwoodson's module is idempotent and will update the configmap if the haproxy-config.template has changed. I don't think there is a reason to delete any longer.

File: ansible/roles/openshift_online_ha_proxy/tasks/main.yml (hunk 1 above)
Comment (themurph) [reply ansible/roles/openshift_online_ha_proxy/tasks/main.yml]: See above: I'm removing it and then I'll re-push /cc @jupierce

**Review by jupierce — APPROVED** [review]

**Pull request merged**